aws/logs_monitoring/enhanced_lambda_metrics.py

```diff
@@ -439,17 +437,34 @@ def parse_metrics_from_report_log(report_log_line):
     Returns:
         metrics - DatadogMetricPoint[]
     """
+    metrics = []
+    tags = ["cold_start:false"]
```

Comment:
Probably a nits, why do you need to front load these two lines before the regex match? The original sequence reflects better to me, because you first check whether the log line is the REPORT log, if not return []...